content/docs/reference/shared-secret.mdx

```diff
@@ -14,33 +13,39 @@ toc_max_heading_level: 2
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Shared Secret
+# Shared Secret Settings
 
-## Summary
+## Shared Secret
 
-**Shared Secret** is the base64-encoded, 256-bit key used to mutually authenticate requests between services. It's critical that secret keys are random, and stored safely.
+**Shared Secret** is the base64-encoded, 256-bit key used to mutually authenticate requests between Pomerium services. It's critical that secret keys are random, and stored safely.
 
-## How to configure
+### How to configure
```

Comment:
I think we should follow https://www.pomerium.com/docs/reference/certificates#certificate-authority as an example here. That is, put `shared_secret` and `shared_secret_file` together as two rows in the same "How to configure" table.

Ditto for signing key.